palm/plugins/dbt/dbt_containerizer.py

```diff
@@ -112,6 +114,21 @@ def write_profile_envs(self) -> None:
                 )
             )
 
+    def get_packages_dir(self) -> str:
+        deps_dir = "dbt_modules"
```

Comment:
If we want to support v1 right away then should we also cover `packages-install-path` now?